### PR TITLE
Preview Banner changes

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #422 :*

- Added Icons to the Preview Banner

- Fixed some of the spacing in the Preview Banner

- Changed Preview Banner Background when there is no image 

- Added Preview Banner messages

-Improved Mobile responsiveness

Desktop View:
image

Mobile Preview: 
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by jadmsaadaot** [comment]
In the description, there is Preview Banner Fix. That does not give any description. I can't understand from it what this PR is about

File: met-web/src/components/engagement/view/PreviewBanner.tsx [hunk 1]
```diff
@@ -55,15 +60,43 @@ export const PreviewBanner = () => {
...
+                            </Grid>
                         </ConditionalComponent>
-                        <Typography variant="body2" minHeight={25}>
-                            Please publish the engagement when ready.
-                        </Typography>
+                        <Grid container alignItems="center" item xs={12}>
+                            <IconButton color="info" aria-label="not published">
+                                <UnpublishedIcon />
+                            </IconButton>
+                            <Typography variant="body2" minHeight={25}>
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/view/PreviewBanner.tsx:96]: This typography has minHeight 25? Can you please show a screenshot of this?

File: met-web/src/components/engagement/view/PreviewBanner.tsx [hunk 2]
```diff
@@ -55,15 +60,43 @@ export const PreviewBanner = () => {
...
         >
             <Grid container direction="row" justifyContent="flex-end" alignItems="flex-start" padding={4}>
-                <Grid item xs={12}>
-                    <Typography variant="h4">Preview Engagement{!isDraft && ' - Published'}</Typography>
+                <Grid item container xs={12}>
+                    <Typography variant="h4" sx={{ mb: 2 }}>
+                        Preview Engagement{!isDraft && ' - Published'}
+                    </Typography>
                     <ConditionalComponent condition={isDraft}>
+                        <ConditionalComponent condition={imageExists === ''}>
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/view/PreviewBanner.tsx:68]: imageExists, the name implies it is a boolean, but here you are checking if it equals a string? 

condition={!savedEngagement.banner_url} ?

File: met-web/src/styles/Theme.ts [hunk 3]
```diff
@@ -37,6 +37,11 @@ export const BaseTheme = createTheme({
         action: {
             active: Palette.action.active,
         },
+        info: {
```
Comment (jadmsaadaot) [inline met-web/src/styles/Theme.ts:40]: if all the same why add dark and light

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Requesting changes for vague description, asking for a screenshot for the minHeight 25 typography and a few cleaner code suggestions

File: met-web/src/components/engagement/view/PreviewBanner.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/PreviewBanner.tsx:96]: image

That was already there before. Here is an image of the code.

File: met-web/src/styles/Theme.ts (hunk 3 above)
Comment (djnunez-aot) [reply met-web/src/styles/Theme.ts:40]: Just thought if we wanted to change it later we have access to it

File: met-web/src/components/engagement/view/PreviewBanner.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/PreviewBanner.tsx:68]: Typescript gave me a boolean error and it asked for me to change it to a string

Comment (djnunez-aot) [reply met-web/src/components/engagement/view/PreviewBanner.tsx:68]: I took this code from somewhere else in the codebase

File: met-web/src/styles/Theme.ts (hunk 3 above)
Comment (jadmsaadaot) [reply met-web/src/styles/Theme.ts:40]: Understood. Let's follow the YAGNI clean code principle for now

File: met-web/src/components/engagement/view/PreviewBanner.tsx (hunk 1 above)
Comment (jadmsaadaot) [reply met-web/src/components/engagement/view/PreviewBanner.tsx:96]: you can do above in a separate PR if it's not straightforward

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**